app/main.py

```diff
@@ -1,3 +1,43 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.customer import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as config_file:
+        config_data = json.load(config_file)
```

Comment:
Do not overload the context manager, place logic not related to the file management outside of its scope.